lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -966,14 +965,14 @@ private static List<Object> decodeRecordReaderStruct(Type type, List<?> fields)
         return newFields;
     }
 
-    public static DataSize writeOrcColumnHive(File outputFile, Format format, CompressionKind compression, Type type, Iterator<?> values)
+    public static void writeOrcColumnHive(File outputFile, Format format, CompressionKind compression, Type type, Iterator<?> values)
```

Comment:
`Remove unused outputFile in OrcTester#writeOrcColumnHive` -> `Remove unused return value`

Reply:
ok